bzl/external/rust/remote/BUILD.alga-0.9.3.bazel

```diff
@@ -0,0 +1,65 @@
+"""
```

Comment:
Ideally it would be nicer to not keep these checked in. This certainly doesn't block this PR though.

Could this directory be `.gitignore`d and these files generated on first run? [This SO answer](https://stackoverflow.com/a/38378377) suggests that it may be possible using remote repoitories.